functions/login_functions.php

```diff
@@ -1,27 +1,36 @@
 <?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);

```

Comment:
Remember to remove debugging code from the PR

Reply:
These were there a long while before I started again on Triniate for PHP 8